src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -51,9 +51,9 @@ object Settings {
             fedoraPassword: String,
             fedoraUrl: URL) = {
     Fedora.setFedoraConnectionSettings(fedoraUrl.toString, fedoraUser, fedoraPassword)
-    new Settings(ownerId = ownerId,
+    new Settings(
       submissionTimestamp = DateTime.parse(submissionTimestamp),
-      bagitDir = bagitDir,
+      depositDir = depositDir,
```

Comment:
Do you really have to change this field and all code that depends on it? You can assign `depositDir.listFiles.find(f => f.isDirectory && f.getName != ".git")` here. Settings is a very convenient object to collect info from command line arguments, property files, system variables and whatever so the run method doesn't have to bother about the environment variations.

By not touching the settings class but just the apply methods (and perhaps the Conf class), we can postpone dependency issues and develop/merge small steps at a time. It would also avoid merge conflicts with parallel development on integrating the license generator.